client/platform/desktop/frontend/components/ImportDialog.vue

```diff
@@ -184,8 +185,8 @@ export default defineComponent({
             prepend-inner-icon="mdi-file-table"
             label="Annotation File (Optional)"
             hint="Optional"
-            @click="openUpload"
-            @click:prepend-inner="openUpload"
+            @click="openUpload('annotation')"
+            @click:prepend-inner="openUpload('annotation')"
```

Comment:
Why prepend-inner?

Reply:
So on desktop we aren't using actual file-inputs, we are using text fields because web file input types really don't work with the direct paths.  This means that they are slightly different than the web file input.   Without the prepend-inner clicking on that icon will just select it.
Web Version:

https://user-images.githubusercontent.com/61746913/150144251-dec36160-cdf5-42dc-97f1-7fd5b7dab24e.mp4


Desktop Version:


https://user-images.githubusercontent.com/61746913/150144278-4901b1d1-5466-40cc-bf68-dc04af6acf7f.mp4